demo-transactions/src/main/resources/api/api.yaml

```diff
@@ -72,6 +72,33 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/AccountCreatedResponse'
+  /api/v1/account/deposit:
+    put:
```

Comment:
Yeah makes sense 🙈  since only deposit of 0 would be idempotent but it doesn't make sense to do it 😅 in the beginning i was using post request but since deposit was updating the entity it felt natural to use a put request i wasn't think of the idempotency.